migrations/4_launch_tokens.js

```diff
@@ -1,11 +1,60 @@
 const WildcardSteward_v3 = artifacts.require("WildcardSteward_v3_matic");
 const ERC721Patronage_v1 = artifacts.require("ERC721Patronage_v1");
+const Dai = artifacts.require("./Dai.sol");
+
+const { ether } = require("@openzeppelin/test-helpers");
+const { daiPermitGeneration } = require("../test/helpers");
 
 const testAccountAddress = "0x8c7A88756EbbF46Ede65E4D678359cAC5f08f7b2";
 
 const starHarbergerTaxRateAnimal = "240" + "0000000000"; // Harberger tax rate of 240% per year
 
+const buyAuctionPermit = async (
```

Comment:
Couldn't get this function to work, very frustraiting... Will let Marvin loose on this :)